pyomo/contrib/gdpopt/GDPopt.py

```diff
@@ -154,10 +590,6 @@ def __exit__(self, t, v, traceback):
 
     def available(self, exception_flag=True):
         """Check if solver is available.
```

Comment:
Why is this comment removed? Should the solver be available even if no subsolvers are? Is there ever going to be a case where GDPOpt is unavailable (as it comes directly with Pyomo)?

Reply:
I think it should always be available, rather than being a function of if the subsolvers are. Without knowing the algorithm in advance of the call to `solve` (which we allow), it's not possible to know *which* subsolvers are necessary. We do fail gracefully when the subsolvers aren't available, so this seems okay to me.